src/worldcereal/openeo/mapping.py

```diff
@@ -76,7 +76,9 @@ def _cropland_map(
     # Postprocess
     if postprocess_parameters.enable:
         if postprocess_parameters.save_intermediate:
-            classes = classes.save_result(format="GTiff")
+            classes = classes.save_result(
+                format="GTiff", options=dict(filename_prefix="cropland-raw")
```

Comment:
also here we need to be careful about hardcoding specific naming schemes. We should take this from the `WorldCerealProductType` otherwise the naming convention loses consistency and gets scattered around.

Reply:
hardcoding of product names avoided in new commit. 
WorldCerealProductType is not an argument in the _cropland_map function, so I did need to refer to WorldCerealProductType.CROPLAND to define the name

Reply:
Makes sense, within `_cropland_map` we can safely assume that that is indeed the product type.